src/fs.ts

```diff
@@ -59,13 +59,13 @@ export class LocalFileSystem implements FileSystem {
 	 * Converts the URI to an absolute path
 	 */
 	protected resolveUriToPath(uri: string): string {
-		return path.resolve(this.rootPath, uri2path(uri));
+		return toUnixPath(path.resolve(this.rootPath, uri2path(uri)));
 	}
 
 	async getWorkspaceFiles(base?: string): Promise<Iterable<string>> {
-		const pattern = base ? path.posix.join(this.resolveUriToPath(base), '**/*.*') : this.resolveUriToPath('file:///**/*.*');
+		const root = base ? this.resolveUriToPath(base) : this.rootPath;
 		const files = await new Promise<string[]>((resolve, reject) => {
-			glob(pattern, { nodir: true }, (err, matches) => err ? reject(err) : resolve(matches));
+			glob('*', { cwd: root, nodir: true, matchBase: true }, (err, matches) => err ? reject(err) : resolve(matches));
```

Comment:
`matchBase` + `*` do the job. `*.*` might not be good in general because it doesn't match files w/o extension (`LICENSE`)